src/index.js

```diff
@@ -1,2 +1,54 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+  return element;
+}
+
+function counter(count = 0) {
```

Comment:
```suggestion
function counter({ count  }) {
```

매개변수도 destructuring 할 수 있다는걸 알고 계신가요?
이렇게 객체 중심으로 작성하는 RORO 패턴이라고 해요.
앞으로 엄청 자주 쓰이게 될테니 어떤 코드를 볼 때 RORO 패턴을 썼는지 파악해 보고, 코드를 작성하면서 RORO 패턴을 적용하면 좋은 상황과 아닌 상황을 구분하는 훈련을 해보시면 좋아요!

참고: https://www.freecodecamp.org/news/elegant-patterns-in-modern-javascript-roro-be01e7669cbd/